File: 프로그래머스/unrated/181839. 주사위 게임 1/주사위 게임 1.java
class Solution {
    public int solution(int a, int b) {
        switch (a % 2 + b % 2)
        {
            case 0 :
                return a < b ? b - a : a - b;
            case 1 :
                return 2 * (a + b);
            case 2 :
                return a * a + b * b;
            default :
                return 1;
        }
    }
}

File: 프로그래머스/0/181839. 주사위 게임 1/주사위 게임 1.js
function solution(a, b) {
    switch (a % 2 + b % 2) {
        case 0 : 
            return Math.abs(a - b);
        case 1 :
            return (a + b) * 2;
        case 2 :
            return a ** 2 + b ** 2;
    }
}